bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/DefaultEntryLogger.java

```diff
@@ -72,8 +78,8 @@
  * the actual ledger entry. The entry log files created by this class are
  * identified by a long.
  */
-public class EntryLogger {
-    private static final Logger LOG = LoggerFactory.getLogger(EntryLogger.class);
+public class DefaultEntryLogger implements EntryLogger {
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultEntryLogger.class);
     static final long UNASSIGNED_LEDGERID = -1L;
     // log file suffix
     static final String LOG_FILE_SUFFIX = ".log";
```

Comment:
I think these should be kept in the `EntryLogger` interface because these are not implementation-specific (UNASSIGNED_LEDGERID - definitely, LOG_FILE_SUFFIX - discussable)